Create testing bundle (#23) (#46)
kyma-project/bundles: bundles/testing-0.0.1/README.md

```
  _______          _    _               
 |__   __|        | |  (_)              
    | |  ___  ___ | |_  _  _ __    __ _ 
    | | / _ \/ __|| __|| || '_ \  / _` |
    | ||  __/\__ \| |_ | || | | || (_| |
    |_| \___||___/ \__||_||_| |_| \__, |
                                   __/ |
                                  |___/ 

```

## Overview

The Testing bundle is a simple bundle which contains one resource - a ConfigMap and a Deployment. It shows all possible parameters.

## Details

> The bundle contains two plans:

Author: The bundle contains the following plans

- minimal - contains only necessary parameters, be aware there is no binding template defined.
- full - contains all possible parameters

> It is a show case how to define chart with minimal and full configuration.

Reviewer: ```suggestion
This bundle is a showcase on how to define a chart with minimal and full configuration.
```
